doc/DOWNLOADS.md

```diff
@@ -105,14 +105,21 @@ In order to upgrade Nicotine+ to a newer release, run the following:
 scoop update nicotine-plus
 ```
 
-### Chocolatey
+### [Chocolatey](https://chocolatey.org/install)
 
-Nicotine+ can be installed using [Chocolatey](https://community.chocolatey.org/packages/nicotine-plus). Run the following:
+Nicotine+ can be installed using [Chocolatey](https://community.chocolatey.org/packages/nicotine-plus). Run the following command:
```

Comment:
Having the header as a link is inconsistent with the rest of the document. The link in the body text should be changed instead:

`Nicotine+ can be installed using [Chocolatey](https://chocolatey.org/install). Run the following:`

Nicotine+ can be installed using [Chocolatey](https://chocolatey.org/install). Run the following:

> Run the following` command`:

If you are going to add this word here, then all of the other similar statements elsewhere need changing also, I don't think this is needed because it's obvious that the sh code blocks are commands that need running.